docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -410,28 +367,18 @@ Vivarium Modeling Strategy
 .. image:: treatment_diagram.svg
    :alt: Compartmental diagram with treatment
 
-For **SAM** 0-6 months infants
-
- - :math:`r2_{ux} = \frac{t}{\text{median time-to-recovery (days) of utx SAM}}` 
- - :math:`t1_{sam} = \frac{t}{\text{median time-to-recovery (days) of tx SAM}}` 
-
-For **SAM** 6-59 months kids
-
- - :math:`r2_{ux} = \frac{t}{\text{median time-to-recovery (days) of utx SAM}}` 
- - :math:`t1_{sam} = \frac{t}{\text{median time-to-recovery (days) of tx SAM}}`
-
-For **MAM** 0-6 months infants
-
- - :math:`r3_{ux} = \frac{t}{\text{median time-to-recovery (days) of utx MAM}}` 
- - :math:`t2_{mam} = \frac{t}{\text{median time-to-recovery (days) of tx MAM}}` 
+So, 
 
-For **MAM** 6-59 months kids
+ - :math:`r2_{ux} = r_{SAM,ux} = \frac{t}{\text{median time-to-recovery (days) of utx SAM}}` 
```

Comment:
You should use the new names for these variables - i.e. `r_{SAM, tx}`, etc.

Reply:
I have both the old and new names currently while I try to figure out how to relabel nicole's figure. I left a to-do note and will update this soon!